libpgmodeler_ui/src/mainwindow.cpp

```diff
@@ -2077,3 +2100,22 @@ void MainWindow::toggleLayersWidget(bool show)
 	layers_wgt->setVisible(show);
 }
 
+QPair <bool, SQLToolWidget *> MainWindow::isAnyManageDbOpened(void)
```

Comment:
As far as I understood this method you need just to check if there's an instance of database being browsed, right? Why return a QPair and where do you need this data structure?

Reply:
There must be a better way I know...
with the qpair I get a handle of the SQLToolWidget to talk to him directly :S